core/src/test/java/org/apache/calcite/rex/RexProgramTest.java

```diff
@@ -2776,6 +2776,11 @@ trueLiteral, literal(1),
     checkSimplify(
         isNotNull(div(vDecimalNotNull(), cast(literal(BigDecimal.valueOf(2.5)), intType))),
         "true");
+
+    checkSimplify(isNull(cast(div(vIntNotNull(), literal(0)), tBigInt())),
```

Comment:
```vIntNotNull``` is valued 0, could we also add non-zero test for it, such as ```vIntNotNull(1)```, just a suggestion.